### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RyanPiao**
1. Wrong URL for Configuration instructions. 
2. Adding URL for Firmware Troubleshooting Guide.

### CODE REVIEW

**Review by danwillm — APPROVED** [review]
lgtm

**Comment by RyanPiao** [comment]
New language(Chinese) added.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ![LucidGloves](https://cdn.discordapp.com/attachments/785135646082990120/873666310855360582/LucidGlovesBlackwbkgnd.png)
 
+**[English](https://github.com/RyanPiao/lucidgloves/blob/main/README.md) | [简体中文](https://github.com/RyanPiao/lucidgloves/blob/main/README_CN.md)**
```
Comment (lucas-vrtech) [inline README.md]: These need to be updated as they link to your fork and not the master repo.

Also, please move this line to the bottom of README.md.

File: README_CN.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+![LucidGloves](https://cdn.discordapp.com/attachments/785135646082990120/873666310855360582/LucidGlovesBlackwbkgnd.png)
+
+**[English](https://github.com/RyanPiao/lucidgloves/blob/main/README.md) | [简体中文](https://github.com/RyanPiao/lucidgloves/blob/main/README_CN.md)**
```
Comment (lucas-vrtech) [inline README_CN.md]: These also need to be updated to the correct repo links (I know the README.CN file doesn't exist here yet)

Comment (lucas-vrtech) [inline README_CN.md]: Also, can you please move this file into a folder called "Readme_Languages"?

**Review by lucas-vrtech — CHANGES_REQUESTED** [review]
Overall looks good, just a few requests. Thanks

File: README_CN.md (hunk 2 above)
Comment (RyanPiao) [reply README_CN.md]: All set.

**Comment by RyanPiao** [comment]
All set.

**Review by lucas-vrtech — APPROVED** [review]
Looks great, thanks!

**Pull request merged**